test/albums/mocking.js

```diff
@@ -0,0 +1,68 @@
+const nock = require('nock');
+
+exports.albumMock = (albumId, albumTitle) => {
+  nock('https://jsonplaceholder.typicode.com')
```

Comment:
try to always import this kind of things from env variables, so that if it changes you can change it everywhere, including tests

Reply:
Do you mean the url of the api?